src/views/questionView.js

```diff
@@ -12,6 +13,7 @@ export const createQuestionElement = (question) => {
 
   // I use String.raw just to get fancy colors for the HTML in VS Code.
   element.innerHTML = String.raw`
+    <p id="${COUNTDOWN_TIME_ID}"></p>
```

Comment:
Ideally we shouldn't include countdown functionality in this branch. How can we solve this?